Envelopes/saull_Peak Envelope Generator.lua

```diff
@@ -0,0 +1,277 @@
+-- @description Peak Envelope Generator
```

Comment:
```suggestion
-- @description Peak envelope generator
```
...to match the convention of native REAPER actions and other scripts. (Same with the filename.)